cypress/helpers/menubar.js

```diff
@@ -18,6 +18,11 @@ export const clickMenubarOptionsButton = () =>
         .contains('Options')
         .click()
 
+export const openDataOptionsModal = () => {
+    clickMenubarOptionsButton()
+    return cy.getBySel('options-menu-list').contains('Data').click()
```

Comment:
I don't think we should use return here. Since Cypress doesn't actually return anything. I know this is probably just a copy paste from the code below (written by @HendrikThePendric originally) but still it should be avoided.